scripts/ci-bazel-test.sh

```diff
@@ -20,6 +20,7 @@ set -o pipefail
 REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 cd $REPO_ROOT
+bazel clean --expunge
```

Comment:
Can we remove this clean now? Did you ever figure out what was causing the test failure previously?

Reply:
Will do, it got fixed after rebase, not sure what exactly was the issue though

Reply:
Done